content/doc/book/blueocean/index.adoc

```diff
@@ -110,68 +124,87 @@ allowfullscreen></iframe>
 </center>
 <br/>
 ++++
-
 endif::[]
 
+
 === Where is the name from?
 
 The name Blue Ocean comes from the book
 link:https://en.wikipedia.org/wiki/Blue_Ocean_Strategy[Blue Ocean Strategy]
-where instead of looking at strategic problems within a contested space you
+where instead of looking at strategic problems within a contested space, you
 look at problems in the larger uncontested space. To put this more simply,
 consider this quote from ice hockey legend Wayne Gretzky: "skate to where the
 puck is going to be, not where it has been".
 
-==== Does Blue Ocean support Freestyle jobs?
+
+==== Does Blue Ocean support freestyle jobs?
 
 Blue Ocean aims to deliver a great experience around Pipeline and be compatible
-with any Freestyle jobs that you have configured in your system. However, they
-won't be able to benefit from any of the features built for Pipelines – for
+with any freestyle jobs you already have configured on your Jenkins instance.
```

Comment:
I am hoping we don't need to ask the Jenkins community about whether or not we can change the word "Freestyle" capitalized as a "proper noun" to just "freestyle" (as a common noun) - see [this Jenkins Community post about the terms "Declarative"- and "Scripted"-Pipeline for context](https://groups.google.com/forum/#!topic/jenkinsci-docs/Zeteb0cEcA8).

Reason, the Jenkins User Documentation scarcely mentions of the term "freestyle" (in fact, this is probably only one of a few pages where "freestyle" is mentioned) and since declarative Pipeline is the way forward, our focus on freestyle documentation is minimal.

Reply:
My thoughts: (weeps). 

I thought "Freestyle" was the right way to write this. Now I know nothing.

Reply:
lol ... this term is used so infrequently and increasingly more so that it's easy for things like this to go unnoticed.